Lib/Shim.php

```diff
@@ -133,6 +133,14 @@ class Shim {
 	 */
 	const VALIDATE_PROPERTY = 'Shim.warnAboutValidateProperty';
 
+	/**
+	 * The name of the shim that checks the usage of deprecated since 3.4.0
+	 * FormHelper::input() and FormHelper::inputs().
+	 *
+	 * @var string
```

Comment:
That we dont need I think.

Reply:
Okay, that was added automatically by IDE.